shell/completion.bash

```
@@ -169,6 +169,10 @@ _sk() {
                     COMPREPLY=($(compgen -W "plain rounded double thick light-double-dashed heavy-double-dashed light-triple-dashed heavy-triple-dashed light-quadruple-dashed heavy-quadruple-dashed quadrant-inside quadrant-outside" -- "${cur}"))
                     return 0
                     ;;
+                --multiline)
+                    COMPREPLY=($(compgen -f "${cur}"))
```

Shellcheck (0.11.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)